docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,377 @@
+
+For Android:
+
+For Android:
+
+UID2Manager.init(
+
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+
+
+
+For iOS:
+
+// Must be set before UID2Manager.shared is accessed UID2Settings.shared.environment = .custom(
+
+  url: URL(string: "https://operator-integ.uidapi.com")!
+
+)
+
+On iOS, you must set UID2Settings before you first access UID2Manager.shared. Any changes made to settings after the first access will not be read.
+
+
+
+Note:
+
+Tokens from the UID2 integration environment are not valid for passing to the bid stream. For the integration environment, you will have different subscription ID and public key values.
+
+
+
+## Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+By default, in the production environment, the UID2 SDK for Android makes API calls to a UID2 server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
+
+
+
+For example, a publisher in Singapore can set the server URL to https://sg.prod.uidapi.com . This is still the UID2 production environment, but the servers are in Singapore.
+
+
+
+For the list of possible base URLs, see Environments.
+
+
+
+A publisher can also set the base URL to https://global.prod.uidapi.com . This URL directs readers to a region geographically close to them, which is ideal if your audience is geographically distributed.
+
+To specify a different UID2 server, you can change it in the init call:
+
+For Android:
+
+UID2Manager.init(
+
+  context = this,
+  serverUrl = " https://global.prod.uidapi.com"
+)
+
+
+
+For iOS:
+
+UID2Settings.shared.environment = .singapore
+
+
+
+// or
+
+
+
+
+
+UID2Settings.shared.environment = .custom(
+
+  url: URL(string: "https://global.prod.uidapi.com")!
+
+)
+
+
+
+
+
+Configure the UID2 Mobile SDK for your mobile application
+
+
+
+After you have instantiated UID2Manager correctly in your mobile app, you will need to pass a UID2 Identity generated from your backend server via Implement Server-Side Token Generation on your backend server section above and pass into the mobile application using the setIdentity method:
+
+
+
+For Android: UID2Manager.getInstance().generateIdentity()
+
+For iOS: UID2Manager.shared.setIdentity()
+
+
+
+## Token Storage
+
+After you call the setIdentity method, the UID2 Identity will be persisted in local file storage.
+
+DANGER
+
+The format of the file stored in the local file storage or the filename itself could change without notice. We recommend that you do not read and update the file directly.
+
+
+
+## Check that the token was successfully generated and then pass it for bid stream use
+
+(**GWH__SW we have to make this (heading) shorter. I wonder can we break it into two steps? Just an idea.**)
```

Comment:
_Pass successfully generated token for bid stream use_

is this shorter?
or even

_Pass generated token for bid stream use_

Reply:
Done (#2), thx.